server/settings.py

```diff
@@ -202,9 +203,40 @@ def env(variable, fallback_value=None):
 # NTB NITF specific behaviour
 # TODO: move this to a module once SD-4650 is fixed
 
-def add_subjects_scheme(subjects):
-    for subject in subjects:
-        subject['scheme'] = 'subject_custom'
+SUBJECT_TYPE = 'tobject.subject.type'
+SUBJECT_MATTER = 'tobject.subject.matter'
+SUBJECT_DETAIL = 'tobject.subject.detail'
+
+
+def build_subject(xml):
```

Comment:
I see it's a temporary sollution, but still this code can be in its own module. imo settings is for config and should import that function